### PR TITLE
Add Ubuntu 15.10 runs

### PR DESCRIPTION
**Pull request by ellismg**
Part of dotnet/corefx#5349

### CODE REVIEW

**Comment by ellismg** [comment]
@dotnet-bot test CI please

**Comment by stephentoub** [comment]
Nice. LGTM.

File: netci.groovy [hunk 1]
```diff
@@ -633,6 +639,7 @@ combinedScenarios.each { scenario ->
                             }
                             break
                         case 'Ubuntu':
+                        case 'Ubuntu15.10':
```
Comment (stephentoub) [inline netci.groovy]: Nit: should we update existing usage of 'Ubuntu' to be 'Ubuntu14.04` or something like that?

Comment (ellismg) [inline netci.groovy]: Probably a good idea, but will require cross repo coordination.

**Comment by ellismg** [comment]
@dotnet-bot test this please

**Comment by ellismg** [comment]
@dotnet-bot test ci please

**Pull request merged**